app/src/main/res/layout/activity_hospital.xml

```diff
@@ -218,6 +218,13 @@
 
     </LinearLayout>
 
+    <TextView
+        android:id="@+id/amountofoverduedevices"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
```

Comment:
View should stand out more against "device_state_overview" view. Maybe add a slightly colored background for the text and a bit of padding?

Reply:
done